os/mm/mm_heap/mm_partition_mgr.c

```diff
@@ -87,28 +99,39 @@ void mm_initialize_ram_partitions(void)
 	 * The overhead value will include the size of mm_heap_s and the
 	 * allocation node and also a 100 byte buffer for future change in mm.
 	 */
-	uint32_t overhead = CONFIG_NUM_APPS * (SIZEOF_MM_ALLOCNODE + sizeof(struct mm_heap_s) + 100); 
+#ifdef CONFIG_MM_PARTITION_HEAP
+	uint32_t overhead = CONFIG_NUM_APPS * (SIZEOF_MM_ALLOCNODE + sizeof(struct mm_heap_s) + 100);
```

Comment:
How about defining extra buffer?

Reply:
Sorry, I dont understand. 
I am just calculating the overhead required to use heap based allocation.

Reply:
oh sorry.. I mean, instead of using 100 directly, how about defining a value for 100?
for example, 
```
#define EXTRA_PADDING 100
```

Reply:
OK. Done.